packages/@aws-cdk/aws-codepipeline-actions/README.md

```diff
@@ -321,6 +321,24 @@ const testAction = new codepipeline_actions.CodeBuildAction({
 });
 ```
 
+#### Cross-Region CodeBuild Actions
```

Comment:
I have tried to make this section as concise as possible to show what is required to implement cross-region build actions. I'm happy to expand this.